ament_cmake_clang_format/ament_cmake_clang_format-extras.cmake

```diff
@@ -18,5 +18,7 @@ find_package(ament_cmake_test QUIET REQUIRED)
 
 include("${ament_cmake_clang_format_DIR}/ament_clang_format.cmake")
 
-ament_register_extension("ament_lint_auto" "ament_cmake_clang_format"
-"ament_cmake_clang_format_lint_hook.cmake")
+if(REGISTER_AMENT_LINT_AUTO_EXTENSIONS)
+  ament_register_extension("ament_lint_auto" "ament_cmake_clang_format"
+    "ament_cmake_clang_format_lint_hook.cmake")
+endif()
```

Comment:
Please keep the newline at the end of the file.

Same below.

Reply:
I didn't see that. Good catch.